fleetactivitytracking/templates/fleetactivitytracking/fatlinkstatisticscorpview.html

```diff
@@ -0,0 +1,42 @@
+{% extends "public/base.html" %}
+{% load bootstrap %}
+{% load staticfiles %}
+{% load i18n %}
+
+{% block title %}Alliance Auth{% endblock %}
+{% block page_title %}{% trans "Fatlink Corp Statistics" %}{% endblock page_title %}
+
+{% block content %}
+    <div class="col-lg-12">
+        <h1 class="page-header text-center">{% blocktrans %}Participation data statistics for {{ month }}, {{ year }}{% endblocktrans %}
+            <div class="text-right">
+                <a href="{% url 'auth_fatlink_view_statistics_corp_month' corpid previous_month|date:"Y" previous_month|date:"m" %}" class="btn btn-info">{% trans "Previous month" %}</a>
+                {%  if next_month %}
+                <a href="{% url 'auth_fatlink_view_statistics_corp_month' corpid next_month|date:"Y" next_month|date:"m" %}" class="btn btn-info">{% trans "Next month" %}</a>
+                {%  endif %}
+            </div>
+        </h1>
+        {% if fatStats %}
+        <table class="table table-responsive">
+            <tr>
+                <th class="col-md-1"></th>
+                <th class="col-md-2 text-center">{% trans "Main Character" %}</th>
+                <th class="col-md-2 text-center">{% trans "Characters" %}</th>
+                <th class="col-md-2 text-center">{% trans "Fats" %}</th>
+                <th class="col-md-2 text-center">{% trans "Average fats" %}</th>
```

Comment:
Worth clarifying what its the average of (with title/tooltip text, maybe) because its not immediately clear unless you look at the code.